File: 백준/Bronze/4796. 캠핑/캠핑.swift
func BOJ_4786(_ L: Int, _ P: Int, _ V: Int) -> Int {
    return V/P*L + min(L, V%P)
}
var count = 1
var result = ""
while let input = readLine(), input != "0 0 0" {
    let LPV = input.split(separator: " ").map { Int($0)! }
    result += "Case \(count): \(BOJ_4786(LPV[0], LPV[1], LPV[2]))\n"
    count += 1
}
print(result)

File: 백준/Bronze/4796. 캠핑/캠핑.py
case_num = 0
answers = []
while True:
    l, p, v = map(int, input().split())
    if l == 0 and p == 0 and v == 0:
      break
    
    case_num += 1

    answer = (v // p) * l + min(l, v % p)
    answers.append(f'Case {case_num}: {answer}')
print('\n'.join(answers))